examples/nexrad/nexrad_copy.py

```diff
@@ -0,0 +1,160 @@
+import logging
+from datetime import datetime, timedelta
+import os
+import sys
+import argparse
+
+from carto.auth import APIKeyAuthClient
+from carto.sql import SQLClient
+from carto.sql import CopySQLClient
+
+import numpy as np
+from siphon.radarserver import RadarServer
+from siphon.cdmr import Dataset
+
+# Example of CopySQLClient usage to stream data from NEXRAD Level 2
+# data to CARTO.
+#
+# For more information see the following URL's:
+#   - https://carto.com/blog/mapping-nexrad-radar-data/
+#   - https://gist.github.com/stuartlynn/a7868cf8ca02931a6408
+#   - http://nbviewer.jupyter.org/gist/dopplershift/356f2e14832e9b676207
+
+# Set up our own logger after cleaning up siphon's one
+logger = logging.getLogger()
+list(map(logger.removeHandler, logger.handlers[:]))
```

Comment:
I guess this is some technicality we don't really want to know about

Reply:
You're right, we don't.

I wanted a logger with a good format. Very useful for development and debugging. But `siphon`  does a call to `basicConfig`:

> This function does nothing if the root logger already has handlers configured for it.

and this code is needed to remove existing handlers (otherwise you get two traces with different handlers/formatters). See https://stackoverflow.com/questions/12034393/import-side-effects-on-logging-how-to-reset-the-logging-module

Another option is to add the logger code before the `siphon` imports, but that is not pep8 compliant.

I guess that, for the shake of clarity, I can use the basic config set up by `siphon`:

```python
import logging
from siphon.radarserver import RadarServer
from siphon.cdmr import Dataset

logger = logging.getLogger('nexrad_copy')
logger.setLevel(logging.INFO)
logger.info('Foo Bar')
```

which produces:
```
INFO:nexrad_copy:Foo Bar
```